### DISCUSSION TITLE
Problem editing homepage

### DISCUSSION THREAD

**Discussion by Karo75** [chamilo/chamilo-lms#5714] [General]
When I open our Chamilo, I see our homepage. But when I want to edit the page, nothing is visible in the editor although you see an image and text on the homepage. How is this possible? What could be the problem? And how could this be solved?

**Comment by Coursenligne** [comment]
Hi,
Can you give inputs on your environment pleas?
Info such as Linux/Windows for server, PHP version, Chamilo version, Fresh or upgrade, dedicated or shared server, multi-URL or not...
Some answer could even contain part of the answer.
